### PR TITLE
v1.4 Documentation: Finalize content and design of UG

### PR DESCRIPTION
**Pull request by ZHTang29**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -8,22 +8,23 @@ the desire to **store**, **access** and **share** your recipes easily?
 
 ***RIZZ***ipe is a **command-based recipe database** that is designed just for chefs _LIKE YOU_ to manage recipes at the touch of a finger!
 
-RIZZipe stars 3 main features:
+***RIZZ***ipe comes equipped with multiple features but stars **3** main ones:
```
Comment (SeeuSim) [inline docs/UserGuide.md:11]: with multiple features, but we love these 3 the most:

File: docs/UserGuide.md [hunk 2]
```diff
@@ -148,17 +152,21 @@ Use our interactive form feature to add your favorite recipe to the recipe book
...
-> ![AddCommandWindow](images/addfCommand.png)
->
-> Tab to move on to the next input field, and shift+tab to go to the previous field!
->
-> A new row automatically pops up when you reach the last row for ingredients and steps, and once you're done just click to the next field!
->
-> Note that multiple tags can be added with commas separators. (e.g. `Vegetarian, Halal`)
-> 
-> Click on Save to add your recipe to the recipe book!
+<div markdown="block" class="alert alert-info">
```
Comment (SeeuSim) [inline docs/UserGuide.md:155]: what's the diff btw block and span

File: docs/UserGuide.md [hunk 3]
```diff
@@ -60,22 +61,22 @@ Recipes are the building blocks of RIZZipe! Each recipe comprises multiple field
 | **Name**        | `n/`   | **`NAME`** <br> `NAME` can contain these characters: `A-Z a-z 0-9 - \ ' " and space`.                                                                       | Yes         | `Chicken Rice`, <br>`Broccoli Stir-Fry`                                                                                                 |
```
Comment (SeeuSim) [inline docs/UserGuide.md:61]: ## Common Terminologies and Shortcuts

Here are some common terminologies and features that we think will help you better utilise the app:

### What is a `Recipe`?

...

### What is a Recipe `Ingredient` ?

...

### Cool! Can I use my keyboard to navigate around?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -276,17 +312,21 @@ Format:
...
+
+* Only full words will be matched i.e. `sub chick` will **not** return stored substitutes for `chicken`
+
+* For inputs with multiple words, it will only match stored ingredients with the same full multi-word input (case-insensitive) i.e. `sub golden syrup` does not return substitutes for `syrup`
+
+* The returned list of ingredients returned will not contain any duplicates.
+
+</div>
 
 ### Clearing the recipe book: `clear`
```
Comment (SeeuSim) [inline docs/UserGuide.md:332]:
```markdown
<div class="alert alert-warning">

⚠️ : This clears your entire `RecipeBook`, and there's no going back!
💡 : If you wish to save your data elsewhere first and start afresh, consider using the [Export](#export) feature first!

</div>
```

**Review by SeeuSim — COMMENTED** [review]
Here's some suggestions, let me know what you think

**Comment by SeeuSim** [comment]
Also, under `### Saving the data:` : 

```markdown

Recipes are saved in a ***[JSON](link)** file at this location:
. (Location where your Recipe Book JAR File `recipebook.jar` is)
├── data
│   └── recipebook.json
├── preferences.json
└── recipebook.jar

```

```

File: docs/UserGuide.md (hunk 3 above)
Comment (filbertphang) [reply docs/UserGuide.md:61]: hm this works. although i think the keyboard shortcuts part isnt necessary, we can just have a one-liner like the current one:
> For experienced users, you may refer to the [Command summary](https://zhtang29.github.io/tp/UserGuide.html#command-summary) and [Keyboard shortcuts summary](https://zhtang29.github.io/tp/UserGuide.html#keyboard-shortcuts-summary) for a summarised table of all the commands and keyboard shortcuts available.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -220,7 +244,7 @@ only those that match any of your specified keywords straight away!
 
 Format:
 `find [PROPERTY] KEYWORD [ADDITIONAL KEYWORDS]...`
-> :bulb: Tip: Supported properties: `name`, `tag`, `ingredient`
+:bulb: Tip: Supported properties: `name`, `tag`, `ingredient`
```
Comment (JamesLiuZX) [inline docs/UserGuide.md:247]: Can add an 'and' to further make it look like an exhaustive list

File: docs/UserGuide.md [hunk 6]
```diff
@@ -276,17 +312,21 @@ Format:
...
-> The search is case-insensitive, i.e. `sub salt` will return all stored substitutions for `Salt` and `salt` across recipes.
->
-> Only full words will be matched i.e. `sub chick` will **not** return stored substitutes for `chicken`
->
-> For inputs with multiple words, it will only match stored ingredients with the same full multi-word input (case-insensitive) i.e. `sub golden syrup` does not return substitutes for `syrup`
->
-> The returned list of ingredients returned will not contain any duplicates.
+<div markdown="block" class="alert alert-info">
+
+* `sub` searches across your stored recipes and within a preloaded suggested substitutions list to provide you with the most accurate and extensive list of substitutions!
```
Comment (JamesLiuZX) [inline docs/UserGuide.md:317]: `within` can be removed.

File: docs/UserGuide.md [hunk 7]
```diff
@@ -256,14 +284,22 @@ Example(s) of usage:
 * `list` followed by `delete 2` deletes the 2nd item stored in the recipe book.
 *  `find chicken` followed by `delete 1` will delete the 1st recipe in the displayed results of the find command.
 
-> Deletes the dish at the specified `INDEX` of the latest list that was displayed.
->
-> The index **must be a positive integer** 1, 2, 3, …
+<div markdown="block" class="alert alert-info">
+
+* Deletes the dish at the specified `INDEX` of the latest list that was displayed.
```
Comment (JamesLiuZX) [inline docs/UserGuide.md:289]: `* Deletes the dish at the specified `INDEX` of the current list.` 
Using past tense`latest list that was displayed` could cause confusion on whether its the previous one or the current one.

File: docs/UserGuide.md [hunk 8]
```diff
@@ -276,17 +312,21 @@ Format:
...
-> The returned list of ingredients returned will not contain any duplicates.
+<div markdown="block" class="alert alert-info">
+
+* `sub` searches across your stored recipes and within a preloaded suggested substitutions list to provide you with the most accurate and extensive list of substitutions!
+
+* :bulb: Tip: Adding a substitution together with an ingredient is recommended since it will boost the number of substitutions listed when that ingredient is queried in the future!
+
+* The search is case-insensitive, i.e. `sub salt` will return all stored substitutions for `Salt` and `salt` across recipes.
+
+* Only full words will be matched i.e. `sub chick` will **not** return stored substitutes for `chicken`
```
Comment (JamesLiuZX) [inline docs/UserGuide.md:323]: `complete words`

File: docs/UserGuide.md (hunk 1 above)
Comment (ZHTang29) [reply docs/UserGuide.md:11]: good suggestion

File: docs/UserGuide.md (hunk 4 above)
Comment (ZHTang29) [reply docs/UserGuide.md:363]: good suggestion, taken

**Pull request merged**